execution_workflows/IsoSCM/bin/postprocess.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+
+import sys
+import argparse
+import csv
+
+def parse_args(args=None):
+    Description = "Reformat IsoSCM bed file into the output file of identification"
+    Epilog = "Example usage: python postprcoess.py <FILE_IN> <RUN_IDENTIFICATION> <IDENTIFICATION_FILE_OUT>"
+
+    parser = argparse.ArgumentParser(description=Description, epilog=Epilog)
+    parser.add_argument("FILE_IN", help="Input IsoSCM output bed file.")
+    parser.add_argument("RUN_MODE", help="Run mode could be 'identification'")
+    parser.add_argument("FILE_OUT", help="Name of output file")
+    return parser.parse_args(args)
+
+
+def parse_input_file(file_in):
+    """
+    This function parses IsoSCM output file to file for
+    identification or relative usage quantification challenge
+    :param file_in: IsoSCM compare step output file to be reformatted to relative usage quantification bed file
+    :return: N/A
+    """
+    outputs = []
+    with open(file_in) as fd:
+        rd = csv.reader(fd, delimiter="\t", quotechar='"')
+        next(fd, None)
+        for row in rd:
+            # obtain proximal PAS
+            chrom = row[2].split(":")[0]
+            # need to do -1 to convert from 1-based gtf file coordinate to 0-based bed file coordinate
+            start = int(row[2].split(":")[1].split("-")[1]) - 1
+            end = start + 1
+            strand = row[8]
+            name = '|'.join([chrom, str(start) + ":" + str(end), strand])
+            output = [chrom, str(start), str(end), name, ".", strand]
+            outputs.append(output)
+
+            # obtain distal PAS
+            # if positive strand, distal PAS is the end of the locus
+            if strand == '+':
+                start = int(row[7].split(":")[1].split("-")[1]) - 1
```

Comment:
Request changing `row[7]` to `row[6]`

Explanation: 
Doing some testing I have found that `locus` is not unique and is shared between some lines. These lines have more than 2 PAS / change points detected in the area. IsoSCM seems to tile pairs of PAS together in quantifying them and `locus` coordinates correspond to the most proximal to most distal region. This means we would grab the same `dPAS` from these lines using the coordinates from `locus` even though it does not correspond to the right distal region coverage in one of the two lines. Here's an example:
![image](https://user-images.githubusercontent.com/58347523/193125924-f8f00d92-3d04-416e-b125-13bbba2cf2dc.png)

The solution is to just grab the dPAS coordinates from the `downstream_segment` column (index 6). They already handle strand so grabbing the coordinate here for dPAS will always be correct. Just grab the last coordinate if `+` and first coordinate if `-` (which you're already doing).